android/src/main/java/co/uk/hive/reactnativegeolocation/location/LocationController.java

```diff
@@ -116,73 +113,43 @@ private boolean hasPermissions() {
     }
 
     private LocationRequest getLocationRequest(CurrentPositionRequest currentPositionRequest) {
-        return LocationRequest.create()
-                .setNumUpdates(1)
-                .setPriority(Priority.PRIORITY_BALANCED_POWER_ACCURACY)
-                .setExpirationDuration(currentPositionRequest.getTimeout());
+        return new LocationRequest.Builder(5000)
+                .setMaxUpdates(1)
+                .setPriority(Priority.PRIORITY_HIGH_ACCURACY)
```

Comment:
Changing this to high accuracy priority as we don't need to monitor the location, we just need to access it once when configuring geofences or changing the home location. We want to be as accurate as possible.